alpine-base-php/php8/etc/php8/conf.d/01_wddx.ini

```diff
@@ -0,0 +1 @@
+extension=wddx.so
```

Comment:
Looks like this extension has been deprecated: https://www.php.net/manual/en/intro.wddx.php and it's not present anymore:

> PHP Warning:  PHP Startup: Unable to load dynamic library 'wddx.so' (tried: /usr/lib/php8/modules/wddx.so (Error loading shared library /usr/lib/php8/modules/wddx.so: No such file or directory), /usr/lib/php8/modules/wddx.so.so (Error loading shared library /usr/lib/php8/modules/wddx.so.so: No such file or directory)) in Unknown on line 0

Reply:
Ah, yes. I had removed the package from the Dockerfile but forgot to remove the config snippet which is separate in the repository.